### PR TITLE
docs:add dimond.js,triangle.js,package.json file(#39515)

### PR DESCRIPTION
**Pull request by pvvarun**

### CODE REVIEW

File: extras/pattern/diamond.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            if(j<=n-i)
+                space=space+" ";
+            else
+                star=star+"* ";
+        }
+        console.log(space,star);
+        star="";
+        space="";
+    }
+    for(let i=1;i<=n;i++){
```
Comment (kartikay-nx) [inline extras/pattern/diamond.js]: @pvvarun If possible please use only two for loop.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "pattern",
+  "version": "1.0.0",
+  "description": "patterns",
+  "main": "index.js",
```
Comment (kartikay-nx) [inline package.json:5]: @pvvarun Add index.js in your directory.

File: extras/pattern/diamond.js (hunk 1 above)
Comment (pvvarun) [reply extras/pattern/diamond.js]: done

File: package.json (hunk 2 above)
Comment (pvvarun) [reply package.json:5]: added index.js

**Pull request merged**